oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/NodeDocumentRevisionCleanerTest.java

```diff
@@ -0,0 +1,549 @@
+package org.apache.jackrabbit.oak.plugins.document;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.SortedMap;
+import java.util.TreeMap;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.apache.jackrabbit.oak.plugins.document.Collection.NODES;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+public class NodeDocumentRevisionCleanerTest {
+
+    @Mock
+    DocumentStore documentStore;
+
+    @Mock
+    DocumentNodeStore documentNodeStore;
+
+    @Mock
+    Checkpoints checkpoints;
+
+    NodeDocument workingDocument;
+
+    NodeDocumentRevisionCleaner nodeDocumentRevisionCleaner;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
This shows up as deprecated. Would using the suggested alternative (`MockitoAnnotations.openMocks` - plus a later `close()`) work?